tensorflow/core/profiler/internal/gpu/BUILD

```diff
@@ -28,6 +30,7 @@ tf_cuda_library(
     cuda_deps = [
         "//tensorflow/core/profiler/internal/gpu:cupti_tracer",
         "//tensorflow/core/profiler/internal/gpu:cupti_wrapper",
+        "//tensorflow/core/profiler/internal/gpu:rocm_tracer",
```

Comment:
doing so means adding quite a few boilerplate codes inside `StreamExecutor`. probably we should still need to do that in case we want to get rid of `-lroctracer64` within `crosstool`.

I'd like to revisit this after I get `rocm_tracer` completely mGPU-aware.